cli/src/account_completion.rs

```diff
@@ -3,7 +3,7 @@
 
 use rustyline::{completion::Completer, Context};
 
-pub struct AccountCompleter;
+pub struct AccountCompleter {}
```

Comment:
God no, just derive Default for this

Reply:
damn, this was only temporary and I forgot to undo it. I wanted to know what's the point of specifying it with { } :joy: